test/v1/services/AmendOtherReliefsServiceSpec.scala

```diff
@@ -99,7 +99,15 @@ class AmendOtherReliefsServiceSpec extends UnitSpec {
         "SERVICE_UNAVAILABLE"              -> InternalError
       )
 
-      input.foreach(args => (serviceError _).tupled(args))
+      val tysErrors = Seq(
+        "INVALID_CORRELATION_ID" -> InternalError,
+        "INVALID_TAX_YEAR"       -> TaxYearFormatError,
+        "INVALID_PAYLOAD"        -> InternalError,
+        "TAX_YEAR_NOT_SUPPORTED" -> RuleTaxYearNotSupportedError,
+        "UNPROCESSABLE_ENTITY"   -> InternalError
+      )
+
+      (errors ++ tysErrors).foreach(args => (serviceError _).tupled(args))
```

Comment:
can we update the body (line 38) to be `  val body: AmendOtherReliefsBody = AmendOtherReliefsBody(None, None, None, None, None, None, None)` since the body isnt actually touched by the test so it is good practise for it to be simplified as not to distract from the purpose of the test

Reply:
Good catch